db/migrations/035_sensitive_field_permission.down.sql

```diff
@@ -0,0 +1,6 @@
+UPDATE api.roles
+SET spec = ROW(
+    (spec).preset_key,
+    array_remove((spec).permissions, 'system:read:sensitive'::api.permission_action)
```

Comment:
`system:read:sensitive` sounds like a name closely related to technical implementation and less relevant to user interaction.

`system:import` might be easier to understand.